cmd/podman/machine/machine.go

```diff
@@ -0,0 +1,30 @@
+package machine
+
+import (
+	"github.com/containers/podman/v3/cmd/podman/registry"
+	"github.com/containers/podman/v3/cmd/podman/validate"
+	"github.com/containers/podman/v3/pkg/domain/entities"
+	"github.com/spf13/cobra"
+)
+
+var (
+	noOp = func(cmd *cobra.Command, args []string) error {
+		return nil
+	}
+	// Command: podman _machine_
+	machineCmd = &cobra.Command{
+		Use:                "machine",
+		Short:              "Manage a virtual machine",
+		Long:               "Manage a virtual machine. Virtual machines are used to run Podman on Macs.",
```

Comment:
s/on Macs//g

Reply:
"Virtual Machines are used to run a remote Podman service to connect to